### PR TITLE
Specs for generate invoice

### PR DESCRIPTION
**Pull request by apoorv1316**
## Summary
Cypress generate invoice specs

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
- [ ] This change requires a documentation update

### Checklist:

- [ ] I have manually tested all workflows
- [ ] I have performed a self-review of my own code
- [ ] I have added automated tests for my code

### CODE REVIEW

**Comment by supriya3105** [comment]
@apoorv1316 Please rebase with develop as the test that is failing with this PR has been fixed. 

@mohinid Please review

**Review by mohinid — CHANGES_REQUESTED** [review]
[data-cy='new-invoice-button'] not found hence tests are failing. Please add it.

**Comment by apoorv1316** [comment]
@mohinid It's fixed now.
Screenshot 2022-08-17 at 9 59 42 AM

File: cypress/support/commands.js [hunk 1]
```diff
@@ -27,3 +29,14 @@ Cypress.Commands.add("loginAsEmployee", function () {
   cy.get(authSelectors.signInButton).click();
   cy.location("pathname").should("eq", "/time-tracking");
 });
+
+Cypress.Commands.add("generateNewInvoice", function(invoice_number){
+  cy.get(invoicesSelector.newInvoiceButton).click()
+  cy.get(invoicesSelector.addClientButton).click()
+  cy.contains("Flipkart").click()
```
Comment (mohinid) [inline cypress/support/commands.js:36]: Why do we have Flipkart as hardcoded value here?

Comment (apoorv1316) [reply cypress/support/commands.js:36]: Just for the sake of simplicity. Flipkart has seed data in which we also have time entries.

**Review by mohinid — APPROVED** [review]
LGTM!

**Pull request merged**